File: Logic Building Assignments/Logic Building Assignment 3/Assignment3_1.c
//  1. Write a program which accept one number from user and print
//     that number of even numbers on screen.

#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;

    //Filter
    if(iNo <= 0)
    {
        return;
    }

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("%d\t",2*iCnt);
    }
    printf("\n");
}

int main()
{
    int iValue = 0;

    printf("How many even numbers do you want : ");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

/*

    Output:
    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 3>gcc Assignment3_1.c

    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 3>a
    How many even numbers do you want : 7
    2       4       6       8       10      12      14

*/

File: Python Assignments/Python Assignment 3/Assignment3_1.py
#   1. Write a program which accept N numbers from user and store it into
#      into List. Return addition of all elements from that List.

def AddElement(Data):
    Sum = 0
    for i in Data:
        Sum = Sum + i
    return Sum

def main():
    size = int(input(("How many elements do you want in the list = ")))
    Arr = list()

    print("Enter a list elements : ")
    for i in range(size):
        No = int(input())
        Arr.append(No)
    print("Your Entered List is : ", Arr)

    Result = AddElement(Arr)
    print("Summation of all elements is : ",Result)

if __name__ == "__main__":
    main()

#   Output: 
#   D:\Python\Marvellous Python\Python Assignments\Python Assignment 3>Python Assignment3_1.py
#   How many elements do you want in the list = 6
#   Enter a list elements :
#   13
#   5
#   45
#   7
#   4
#   56
#   Your Entered List is :  [13, 5, 45, 7, 4, 56]
#   Summation of all elements is :  130